src/vm/peimage.inl

```diff
@@ -481,8 +481,19 @@ inline BOOL PEImage::HasNativeHeader()
         return GetLoadedLayout()->HasNativeHeader();
     else
     {
-        PEImageLayoutHolder pLayout(GetLayout(PEImageLayout::LAYOUT_ANY,LAYOUT_CREATEIFNEEDED));
```

Comment:
I'm living in North Carolina.  I suppose local vernacular would be what y'all prefer.